Add more documentation (#32)
QwantResearch/addresses-importer: importers/openaddresses/README.md

# OpenAddresses importer

> This importer import the addresses from [OpenAddresses]. It requires as first argument, the folder

Reviewer: ```suggestion
This is the importer for [OpenAddresses]. It requires as first argument, the folder
```
